.spdx-ignore

```diff
@@ -0,0 +1,41 @@
+LICENSE
+*.html
+*.json
+*.md
+*.png
+*.svg
+*.rst
+*.txt
+*.ptx
+*.pyc
+.gitattributes
+.gitignore
+.git-blame-ignore-revs
```

Comment:
We added SPDX identifiers to these files since they support comments. We can remove them from the exclusions.